source/io/index.rst

```diff
@@ -119,19 +119,19 @@ premiers caractères d'un fichier dans un autre fichier:
 .. literalinclude:: ./examples/stream.py
    :linenos:
 
-io.StringIO
------------
+:py:class:`io.StringIO`
+------------------------
 
 .. code-block:: python
 
     io.StringIO(initial_value='', newline='\n')
 
-Permet d'ouvrir un flux textuel en mémoire. Le buffer est vidé lorsqu'on close() le flux.
-io.StringIO hérite de io.BaseIO. Lorsque qu'on appelle open(mode='t'), python crée, de façon caché, un io.StringIO.
+Permet d'ouvrir un flux textuel en mémoire. Le buffer est vidé lorsqu'on ``close()`` le flux.
+:py:class:`io.StringIO` hérite de :py:class:`io.IOBase`. Lorsque qu'on appelle ``open(mode='t')``, python crée, de façon caché, un :py:class:`io.StringIO`.
 
-initial_bytes: Permets d'avoir des données à la création.
+``initial_value``: Permets d'avoir des données à la création.
 
-newline: Spécifie comment les sauts de ligne fonctionnent. Valeurs
+``newline``: Spécifie comment les sauts de ligne fonctionnent. Valeurs
 possible: None, '', '\\n'(par défaut), '\\r' ou '\\r\\n'.
```

Comment:
De ce que j'ai lu,  '\r'  ne fait pas de saut à la ligne mais remet les curseurs au début de la ligne. Alors que les deux autres vont sur une nouvelle ligne.

- https://stackoverflow.com/questions/1761051/difference-between-n-and-r
- https://stackoverflow.com/questions/15433188/r-n-r-n-what-is-the-difference-between-them